nx-cloud/templates/cloud.yml

```diff
@@ -974,3 +975,59 @@ spec:
   ports:
     - port: 8080
       targetPort: 8080
+---
```

Comment:
should we add a flag in the helm values file to enable these? Just so we don't enable on prod until we know it's ready 🤔  What do you think?